Postr/src/com/postr/DataTypes/Outputs/BaseFeed.java

```diff
@@ -83,14 +83,14 @@ private long WritePost(String postSubject, String postContents,
 
 	private List<String> getTags(LinkSet links) {
 		List<String> tagsForPosting;
+		if(defaultTags != null){
```

Comment:
This won't work if you have both default tags and actual tags, because the default tags will be overridden by the actual tags.

Got 5 minutes to fix this?  Or shall I pick it up?